src/contracts/atlas/Escrow.sol

```diff
@@ -241,6 +242,53 @@ abstract contract Escrow is AtlETH {
         return (result, gasLimit);
     }
 
+    function _getBidAmount(
+        address environment,
+        SolverOperation calldata solverOp,
+        bytes memory dAppReturnData,
+        EscrowKey memory key
+    )
+        internal
+        returns (uint256 bidAmount)
+    {
+        // NOTE: To prevent a malicious bundler from aggressively collecting storage refunds,
+        // solvers should not be on the hook for any 'on chain bid finding' gas usage.
```

Comment:
How does a bundler do this if the ops relay only gives a certain number of solverOps to the bundler and only when they are profitable? Is it by using too small of a gas limit?
I feel better if solvers have to cover all gas for on chain bid finding including calldata, or else we open ourselves up more for spam and have to rely more on the reputation to prevent it

Reply:
ops relay is trusted in that scenario.

If ops relay = bundler, they're incentivized to just cram as many expected-to-fail operations as possible into the tx, along with one successful one.  Hard to monitor and measure the ops relay's honesty on this subject.

That being said, in the current version we do make each solver responsible for the gas cost of finding their bids.  I'll edit the note.